tools/links.py

```diff
@@ -8,16 +8,16 @@
     python links.py
 """
 
-import os
+from pathlib import Path
 import re
 
-ROOT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+ROOT_DIR = Path(Path(Path.resolve(Path(__file__)).parent).parent)

```

Comment:
No need for path-nesting. You can simply do:

```suggestion
ROOT_DIR = Path(__file__).parent.parent
```